src/main/java/sirius/biz/mongo/PrefixSearchableEntity.java

```diff
@@ -39,9 +40,16 @@
     @NoJournal
     private final StringList searchPrefixes = new StringList();
 
+    @Transient
+    private boolean forceUpdateSearchPrefixes;
+
     @BeforeSave
     protected void updateSearchField() {
-        if (!isNew() && !isAnyMappingChanged()) {
+        if (!isNew()
+            && !forceUpdateSearchPrefixes
+            && getDescriptor().isFetched(this,
```

Comment:
why?

Reply:
> Furthermore, the searchPrefixes field is created if it didn't exist already.

Reply:
but that hasn't todo anything with fetching it in a query?!

Reply:
it only checks if the field is present or not